src/limbo/experimental/model/ssgpr.hpp

```diff
@@ -32,10 +33,10 @@ namespace limbo {
                 {
                     assert(_dim_in > 0 && _dim_out > 0);
 
-                    int n = _mapping.dim_out();
-                    _L = _sigma_n * _sigma_n * Eigen::MatrixXd::Identity(n, n);
-                    _B = Eigen::MatrixXd::Zero(n, _dim_out);
-                    _W = Eigen::MatrixXd::Zero(n, _dim_out);
+                    _dim_mapping = _mapping.dim_out();
+                    _L = _sigma_n * Eigen::MatrixXd::Identity(_dim_mapping, _dim_mapping);
```

Comment:
Why did you remove the _sigma_n^2? I don't really mind, but just to know..

Reply:
This is from I-SSGPR paper section 3.3, where they say R0 should be the cholesky factor of sigma_n^2 eye(D).

Reply:
> This is from I-SSGPR paper section 3.3, where they say R0 should be the cholesky factor of sigma_n^2 eye(D).

True dat..